### ISSUE TITLE
Iterable is not defined 

### ISSUE THREAD

**Issue by kasperpeulen** [gajus/eslint-plugin-flowtype#105]
Not sure if this is an issue for this plugin, but eslint is complaining about Iterable to be not defined:

![image](https://cloud.githubusercontent.com/assets/1035299/18421109/0c4113a8-7882-11e6-9f31-929bf858601c.png)

**Comment by danharper** [comment]
To your `.eslintrc` add the following:

```
"globals": {
  "Generator": true,
  "Iterator": true,
  "Iterable": true,
  "$Keys": true,
  "$Shape": true,
},
```

we can probably expose an eslint environment in the plugin, adding these automatically.

**Comment by danharper** [comment]
Or, if you enable the `define-flow-type` rule, all Flow types get marked as defined.
